core/metadata/test/fixtures/modules/optional-inject-module/OptionalInjectService.ts

```
@@ -0,0 +1,13 @@
+import { Inject, InjectOptional, SingletonProto } from '@eggjs/core-decorator';
+
+interface PersistenceService {
+}
```

Biome

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)